### ISSUE TITLE
Can't click on compound drawable inside AutoCompleteTextView

### ISSUE THREAD

**Issue by renas** [RobotiumTech/robotium#588]
_From [milk...@gmail.com](https://code.google.com/u/108558634670799200493/) on August 29, 2013 04:36:02_

I would like to suggest a feature which could make it possible to click inside AutoCompleteTextView's compound drawable. I have no option to click on a drawable which is placed inside AutoCompleteTextView.

_Original issue: http://code.google.com/p/robotium/issues/detail?id=509_

**Comment by renas** [comment]
_From [renasr...@gmail.com](https://code.google.com/u/108052454046732184536/) on August 29, 2013 10:59:32_

Would it be possible for you to share an apk that has a drawable placed inside AutomCompleteTextView?

**Owner:** renasr...@gmail.com

**Comment by renas** [comment]
_From [milk...@gmail.com](https://code.google.com/u/108558634670799200493/) on August 30, 2013 00:16:43_

Unfortunately I'm not able to provide you the apk. Basicly, i have an AutoCompleteTextView component on which I have set compound drawable on the right side (search or clear images depending on situation). It would be great if you provide a way of assert for the existance of that compound drawable and also a way to click on it.

For the click I found a dirty solution, i got coordinates for the AutoCompleteTextView and calculated where the drawable should be and performed clickOnScreen, but that is not very elegant.
